x-pack/test/banners_functional/tests/index.ts

```diff
@@ -9,7 +9,6 @@ import { FtrProviderContext } from '../ftr_provider_context';
 
 export default function ({ loadTestFile }: FtrProviderContext) {
   describe('banners - functional tests', function () {
-    loadTestFile(require.resolve('./global'));
```

Comment:
This should be reverted once https://github.com/elastic/kibana/pull/140688 get merged

Reply:
Is it not a duplicate?

Reply:
No. https://github.com/elastic/kibana/pull/140688 only fixes a bug and re-enable the FTR tests. Your PR migrates the test data to stop using esArchiver.

Reply:
Oh, the test you mean? No, the two suites are testing different things. One only uses global banner configuration, the other one test the behavior when both global config on a given space's config are set.